moveit/src/components/ExperienceBar.tsx

```diff
@@ -0,0 +1,17 @@
+
+
+export function ExperienceBar() {
```

Comment:
TOC da minha parte 🤓 
```suggestion
export function ExperienceBar() {
```